buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -299,7 +299,7 @@
       },
       "inputs": {
         "filename": "docker",
-        "arguments": "run --rm $(DockerCommonRunArgs_Ubuntu1404) $(PB_GitDirectory)/Tools/msbuild.sh $(PB_GitDirectory)/src/pkg/packaging/dir.proj $(AdditionalMSBuildPackagingArguments) $(DistroSpecificMSBuildArguments) /p:OutputRid=ubuntu.16.04-$(PB_TargetArchitecture) /p:BuildRuntimeDebs=false",
+        "arguments": "$(CommonDockerCommandToBuildRuntimeDepDebPackage) /p:OutputRid=ubuntu.16.04-$(PB_TargetArchitecture)",
```

Comment:
Refactoring argument list used for packaging and publishing Debian Runtime deps .